app/Http/Controllers/Three/SignupsController.php

```diff
@@ -102,6 +102,12 @@ public function index(Request $request)
      */
```

Comment:
@DFurnes I'm a little confused how to use query scope on this. I tried to do something like 
```
$northstarId = getNorthstarId($request);
Signup::scopeWhereVisible($query, $northstarId)->get();
```
But not sure how we can do that here because it sends the whole `$query` to `$this->paginatedCollection` and does more work later to find the exact results we're querying for? Am I thinking of this incorrectly?

Reply:
You should be able to access the scopes here since it's a `Signup` query:

```php
$query = $this->newQuery(Signup::class);

// ...

$query = $query->whereVisible($northstarId);
```

Reply:
ohhhhhh that makes a lot more sense! Thank you!!